style.css

```diff
@@ -0,0 +1,293 @@
+/* Global styles */
+
+* {
+  padding: 0%;
+  margin: 0%;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Poppins', sans-serif;
+  font-size: 10px;
+}
+
+a {
+  text-decoration: none;
+  color: #1a2236;
+}
+
+/*      HEADER   */
+#header .nav-bar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 35px;
+  padding: 20px;
+  border-bottom: 0.1px solid gray;
+}
+
+#header .nav-bar .hello {
+  font-size: 2rem;
+}
+
+#header .nav-bar .about {
+  font-size: 2rem;
+}
+
+#header .nav-bar .portfolio {
+  font-size: 2rem;
+}
+
+#header .nav-bar .contact {
+  font-size: 2rem;
+}
+
+#header .nav-bar .hello:hover {
+  font-size: 2.1rem;
+  text-decoration: underline;
+}
+
+#header .nav-bar .about:hover {
+  font-size: 2.1rem;
+  text-decoration: underline;
+}
+
+#header .nav-bar .portfolio:hover {
+  font-size: 2.1rem;
+  text-decoration: underline;
+}
+
+#header .nav-bar .contact:hover {
+  font-size: 2.1rem;
+  text-decoration: underline;
+}
+
+/* Container Left & Right */
+#body {
+  height: 100vh;
+  width: 100%;
+}
+
+#body .container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
- [ ] You did a tremendous work on designing the first section of you portfolio but please stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9) as it was part of [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md).

Here you have a screenshot of your project:

![image](https://user-images.githubusercontent.com/74506933/154732073-7d5c5929-9979-46bf-9a51-69d1b8dc13ba.png)

Here you have a screenshot of the design:

![image](https://user-images.githubusercontent.com/74506933/154732184-dbbe4452-cc14-41bc-ad6e-4a9f2cf1bc79.png)